### PR TITLE
Separate dataSpec and modelSpec from job.json

### PR DESCRIPTION
**Pull request by alexandruuBytex**
## Description

The dataSpec and modelSpec data from the job.json is placed in 2 separate files dataSpec.json and modelSpec.json.
In the job.son file, the paths to the other 2 files are present.

## Type of Change

- [ ] Bug Fix
- [ ] New Feature
- [ ] Breaking Change
- [ X ] Refactor
- [ ] Documentation
- [ ] Other (please describe)

## Checklist

- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md)
- [ ] Existing issues have been referenced (where applicable)
- [ ] I have verified this change is not present in other open pull requests
- [ ] Functionality is documented
- [ ] All code style checks pass
- [ ] New code contribution is covered by automated tests
- [ ] All new and existing tests pass

### CODE REVIEW

**Comment by elqurio** [comment]
The SDK changes no longer seem to be a part of this PR. Should I add them?

File: scripts/runMinikube.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```
Comment (myungjin) [inline scripts/runMinikube.sh:1]: Why is this file in this PR? This was already merged into the main branch. Please remove it.

File: examples/parallel_experiment/parallel_experiment.zip
Comment (myungjin) [inline examples/parallel_experiment/parallel_experiment.zip:1]: Why does this file need to be updated? Does your change require code update as well? If so, why do other examples not have zip files updated?

File: lib/python/flame/examples/async_mnist/aggregator/config.json [hunk 2]
```diff
@@ -2,6 +2,10 @@
     "taskid": "49d06b7526964db86cf37c70e8e0cdb6bd7aa742",
     "backend": "p2p",
     "brokers": [
+        {
```
Comment (myungjin) [inline lib/python/flame/examples/async_mnist/aggregator/config.json:5]: This is also already included in the main branch. This is not the change that should be included in this PR.

File: lib/python/flame/examples/async_mnist/trainer/config1.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "backend": "p2p",
     "brokers": [
         {
-            "host": "broker.hivemq.com",
+            "host": "localhost",
```
Comment (myungjin) [inline lib/python/flame/examples/async_mnist/trainer/config1.json:6]: same here!

File: lib/python/flame/examples/async_mnist/trainer/config2.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
     "backend": "p2p",
     "brokers": [
         {
-            "host": "broker.hivemq.com",
+            "host": "localhost",
```
Comment (myungjin) [inline lib/python/flame/examples/async_mnist/trainer/config2.json:6]: same here

File: lib/python/flame/examples/async_mnist/trainer/config3.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
     "backend": "p2p",
     "brokers": [
         {
-            "host": "broker.hivemq.com",
+            "host": "localhost",
```
Comment (myungjin) [inline lib/python/flame/examples/async_mnist/trainer/config3.json:6]: same here

File: lib/python/flame/examples/async_mnist/trainer/config4.json [hunk 6]
```diff
@@ -3,7 +3,7 @@
     "backend": "p2p",
     "brokers": [
         {
-            "host": "broker.hivemq.com",
+            "host": "localhost",
```
Comment (myungjin) [inline lib/python/flame/examples/async_mnist/trainer/config4.json:6]: same here

**Review by myungjin — CHANGES_REQUESTED** [review]
This PR contains the changes which were already merged in the main branch. It can't be approved as is.

File: examples/parallel_experiment/parallel_experiment.zip
Comment (alexandruuBytex) [reply examples/parallel_experiment/parallel_experiment.zip:1]: Only the name of the file have been changed in order to work with the runExample script. The zip file must have the same name as the example.

File: lib/python/flame/examples/async_mnist/aggregator/config.json (hunk 2 above)
Comment (alexandruuBytex) [reply lib/python/flame/examples/async_mnist/aggregator/config.json]: I removed the change.

File: lib/python/flame/examples/async_mnist/trainer/config4.json (hunk 6 above)
Comment (alexandruuBytex) [reply lib/python/flame/examples/async_mnist/trainer/config4.json]: I removed the change.

File: scripts/runMinikube.sh (hunk 1 above)
Comment (alexandruuBytex) [reply scripts/runMinikube.sh]: The file have been removed.

**Review by elqurio — APPROVED** [review]

**Review by openwithcode — APPROVED** [review]

**Review by openwithcode — APPROVED** [review]

**Review by openwithcode — APPROVED** [review]

**Pull request closed without merging**